lib/redux/middleware/sentry_api_middleware.dart

```diff
@@ -17,6 +17,8 @@ class SentryApiMiddleware extends MiddlewareClass<AppState> {
   dynamic call(Store<AppState> store, action, next) {
     if (action is FetchOrgsAndProjectsAction) {
       final thunkAction = (Store<AppState> store) async {
+        final transaction =
+            Sentry.startTransaction('FetchOrgsAndProjectsAction', 'task');
```

Comment:
I'm not entirely sure a transaction makes sense here, I'd rather try to create a span out of the transaction bound to the scope

Reply:
Yeah, we might run into the case that first render is done before orgs and projects are synced. Also, I'm kinda confused as to what to choose for name and operation, the flutter docs don't really go into this in detail.

Reply:
we can discuss about that today over our call, but naming things is hard, do not bother about that right now